# Copyright (C) 1999 Free Software Foundation, Inc.
# Valek Filippov <frob@df.ru>, 1999.
#
msgid ""
msgstr ""
"Project-Id-Version: gnome-db\n"
"POT-Creation-Date: 2002-01-25 01:10+0100\n"
"PO-Revision-Date: 2001-02-19 12:30+03:00\n"
"Last-Translator: Valek Filippov <frob@df.ru>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=koi8-r\n"
"Content-Transfer-Encoding: 8-bit\n"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: lib/gda-client/gda-connection-pool.c:268
#, c-format
msgid "could not open connection to %s"
msgstr "не удалось открыть соединение с %s"

# fe/fe_main.c:205
#: lib/gda-client/gda-connection-pool.c:281
#, c-format
msgid "Data source %s not found"
msgstr "Не найден источник данных %s"

#: lib/gda-client/gda-connection.c:433
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "База данных '%s' не найдена в конфигурации системы"

#: lib/gda-client/gda-connection.c:437 lib/gda-client/gda-connection.c:454
msgid "[GDA Client Library]"
msgstr ""

#: lib/gda-client/gda-connection.c:453
msgid "Could not open CORBA factory"
msgstr ""

# gda-clnt/gda-field.c:95
#: lib/gda-client/gda-field.c:243 lib/gda-client/gda-field.c:298
#: lib/gda-client/gda-field.c:307
#: providers/gda-mysql-server/gda-mysql-recordset.c:197
msgid "<NULL>"
msgstr "<NULL>"

# gda-clnt/gda-field.c:101 gda-clnt/gda-field.c:103
#: lib/gda-client/gda-field.c:248 lib/gda-client/gda-field.c:251
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Неизвестный GDA тип(NULL)>"

#: lib/gda-client/gda-field.c:264
#, fuzzy
msgid "true"
msgstr "Истина"

# fe/fe_main.c:249
#: lib/gda-client/gda-field.c:266
#, fuzzy
msgid "false"
msgstr "Ложь"

#: lib/gda-common/gda-corba.c:88
#, c-format
msgid "CORBA System exception: %s"
msgstr "Исключение системы CORBA: %s"

#: lib/gda-common/gda-corba.c:123
msgid "True"
msgstr "Истина"

# fe/fe_main.c:249
#: lib/gda-common/gda-corba.c:124
msgid "False"
msgstr "Ложь"

#: lib/gda-common/gda-error.c:128
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: Сервер не отвечает."

#: lib/gda-common/gda-error.c:135
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Возникла ошибка в системе CORBA."

#: lib/gda-common/gda-error.c:152
#, fuzzy
msgid "[CORBA User Exception]"
msgstr "Исключение пользователя CORBA %s"

#: lib/gda-common/gda-error.c:201
msgid "Unknown CORBA exception for connection"
msgstr ""

#: lib/gda-common/gda-init.c:55
msgid "Attempt to initialize an already initialized client"
msgstr "Попытка инициализировать уже инициализированного клиента"

#: lib/gda-common/gda-init.c:64 lib/gda-server/gda-server-init.c:121
#, fuzzy
msgid "Could not initialize Bonobo"
msgstr "Не удалось инициализировать ct_diag()"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Сохранение журнала...x"

#. compose message
#: lib/gda-common/gda-log.c:68
msgid "ERROR: "
msgstr "ОШИБКА:"

#: lib/gda-common/gda-log.c:68
msgid "MESSAGE: "
msgstr "СООБЩЕНИЕ:"

#: lib/gda-common/gda-thread.c:168
msgid "thread is already running"
msgstr "thread уже запущен"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: lib/gda-common/gda-util.c:113
#, fuzzy, c-format
msgid "Could not open file %s"
msgstr "Не удалось установить локализацию для '%s'"

# fe/fe_log.c:92
#: lib/gda-common/gda-util.c:138
#, fuzzy, c-format
msgid "Could not create file %s"
msgstr "не удалось получить поле"

#: lib/gda-common/gda-xml-database.c:166
msgid "bad formed document"
msgstr ""

#: lib/gda-common/gda-xml-database.c:267
#, c-format
msgid "table %s already exists"
msgstr "таблица %s уже существует"

#: lib/gda-common/gda-xml-database.c:449
#, c-format
msgid "%p is not a valid table"
msgstr "%p не является допустимой таблицей"

# fe/fe_main.c:205
#: lib/gda-common/gda-xml-item.c:386
#, fuzzy, c-format
msgid "Item with id %s not found"
msgstr "Не найден источник данных %s"

#: lib/gda-server/gda-server-connection.c:404
#, fuzzy
msgid "Connection being closed"
msgstr "Видимо соединение умерло"

#: lib/gda-server/gda-server-connection.c:407
msgid "This connection is being closed, so all listeners are released"
msgstr ""

#: lib/gda-server/gda-server-error.c:41
msgid "Could not get pointer to server implementation"
msgstr "Невозможно получить указатель на реализацию сервера"

#: lib/gda-server/gda-server-init.c:43 lib/gda-server/gda-server-init.c:49
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Получен сигнал %d, выключение"

#: lib/gda-server/gda-server-init.c:53
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Получен сигнал %d. Игнорируется"

#: lib/gda-server/gda-server-init.c:104
msgid "Attempt to initialize an already initialized provider"
msgstr "Попытка инициализации уже инициализированного провайдера"

#: lib/gda-server/gda-server-recordset.c:52
#: lib/gda-server/gda-server-recordset.c:60
#: lib/gda-server/gda-server-recordset.c:100
#: lib/gda-server/gda-server-recordset.c:108
#, c-format
msgid "%s not implemented"
msgstr "%s не реализовано"

#: lib/gda-server/gda-server.c:128
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Запуск провайдера %s с не реализованными функциями"

#: lib/gda-server/gda-server.c:214
#, c-format
msgid "CORBA system exception %s"
msgstr "Исключение системы CORBA %s"

#: lib/gda-server/gda-server.c:218
#, c-format
msgid "CORBA user exception: %s"
msgstr "Исключение пользователя CORBA %s"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-default-server/GNOME_GDA_Provider_Default.oaf.in.h:1
#, fuzzy
msgid "Default provider connection factory"
msgstr "не удалось открыть соединение с %s"

#: providers/gda-default-server/GNOME_GDA_Provider_Default.oaf.in.h:2
msgid "GDA Default Datasource Access"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:127
msgid "You must specify a directory for the database to be stored"
msgstr ""

#: providers/gda-default-server/gda-default-connection.c:235
#: providers/gda-mysql-server/gda-mysql-connection.c:283
#: providers/gda-oracle-server/gda-oracle-connection.c:334
#: providers/gda-postgres-server/gda-postgres-connection.c:486
#, fuzzy
msgid "Unknown schema type"
msgstr "Неизвестный тип ORACLE %ld"

#: providers/gda-default-server/main-default.c:70
#, fuzzy
msgid "Could not register Default provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера Mail"

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:1
msgid "GDA Datasource Access for LDAP"
msgstr ""

#: providers/gda-ldap-server/GNOME_GDA_Provider_LDAP.oaf.in.h:2
msgid "LDAP provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:331
#: providers/gda-ldap-server/gda-ldap-connection.c:293
#: providers/gda-mdb-server/gda-mdb-connection.c:372
#: providers/gda-mysql-server/gda-mysql-connection.c:490
#: providers/gda-odbc-server/gda-odbc-connection.c:624
#: providers/gda-odbc-server/gda-odbc-connection.c:657
#: providers/gda-oracle-server/gda-oracle-connection.c:479
#: providers/gda-postgres-server/gda-postgres-connection.c:767
#: providers/gda-sybase-server/gda-sybase-error.c:100
#: providers/gda-sybase-server/gda-sybase-error.c:118
#: providers/gda-tds-server/gda-tds-error.c:128
#, c-format
msgid "error '%s' at %s"
msgstr "ошибка '%s' в %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:337
#: providers/gda-ldap-server/gda-ldap-connection.c:298
#: providers/gda-ldap-server/gda-ldap-connection.c:299
#: providers/gda-mdb-server/gda-mdb-connection.c:377
#: providers/gda-mysql-server/gda-mysql-connection.c:498
#: providers/gda-mysql-server/gda-mysql-connection.c:499
#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:661
#: providers/gda-oracle-server/gda-oracle-connection.c:484
#: providers/gda-oracle-server/gda-oracle-connection.c:485
#: providers/gda-postgres-server/gda-postgres-connection.c:775
#: providers/gda-postgres-server/gda-postgres-connection.c:776
#: providers/gda-primebase-server/gda-primebase-connection.c:375
#: providers/gda-primebase-server/gda-primebase-connection.c:376
#: providers/gda-sybase-server/gda-sybase-error.c:112
#: providers/gda-tds-server/gda-tds-error.c:137
#: providers/gda-tds-server/gda-tds-error.c:138
msgid "Not available"
msgstr "Не доступно"

#: providers/gda-interbase-server/gda-interbase-connection.c:338
#: providers/gda-ldap-server/gda-ldap-connection.c:300
#: providers/gda-mdb-server/gda-mdb-connection.c:378
#: providers/gda-mysql-server/gda-mysql-connection.c:500
#: providers/gda-oracle-server/gda-oracle-connection.c:486
#: providers/gda-postgres-server/gda-postgres-connection.c:777
#: providers/gda-primebase-server/gda-primebase-connection.c:377
#: providers/gda-sybase-server/gda-sybase-error.c:113
#: providers/gda-tds-server/gda-tds-error.c:139
msgid "error"
msgstr "ошибка"

#: providers/gda-ldap-server/main-ldap.c:73
msgid "Could not register LDAP provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера LDAP"

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:1
msgid "GDA Datasource Access for Interbase"
msgstr ""

#: providers/gda-interbase-server/GNOME_GDA_Provider_InterBase.oaf.in.h:2
msgid "InterBase provider connection factory"
msgstr ""

#: providers/gda-interbase-server/gda-interbase-connection.c:327
#, fuzzy, c-format
msgid "Error code %ld (%s)"
msgstr "Код ошибки %ld"

#: providers/gda-interbase-server/main-interbase.c:74
msgid "Could not register INTERBASE provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера INTERBASE"

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:1
msgid "GDA Datasource Access for Mail"
msgstr ""

#: providers/gda-mail-server/GNOME_GDA_Provider_Mail.oaf.in.h:2
msgid "Mail provider connection factory"
msgstr ""

#: providers/gda-mail-server/gda-mail-connection.c:107
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-sybase-server/gda-sybase-connection.c:313
#: providers/gda-tds-server/gda-tds-connection.c:283
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "Необработанный SCHEMA_QTYPE %d"

#: providers/gda-mail-server/main-mail.c:74
msgid "Could not register Mail provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера Mail"

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:1
msgid "GDA Datasource Access for MDB (MS Access) files"
msgstr ""

#: providers/gda-mdb-server/GNOME_GDA_Provider_MDB.oaf.in.h:2
msgid "MDB provider connection factory"
msgstr ""

# fe/fe_editor.c:185
#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:144
#: providers/gda-sybase-server/gda-sybase-command.c:167
#: providers/gda-tds-server/gda-tds-command.c:125
#, c-format
msgid "executing command '%s'"
msgstr "выполнение команды \"%s\""

# fe/fe_editor.c:185
#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:149
#, c-format
msgid "info '%s'"
msgstr "информация \"%s\""

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Вы не можете указывать UNIX_SOCKET, если вы также указываете Хост или "
"Порт. Пожалуйста удалить параметры Хост и Порт или UNIX_SOCKET"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s:%d: Вы должны ввести имя файла базы данных MS Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr ""
"Открытие соединения с параметрами имя_файла=%s, пользователь=%s, пароль=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:427
msgid "mysql_recset is NULL"
msgstr "mysql_recset имеет значение NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:469
#: providers/gda-mysql-server/gda-mysql-connection.c:845
#: providers/gda-sybase-server/gda-sybase-connection.c:494
#: providers/gda-tds-server/gda-tds-connection.c:453
msgid "No table name given for SCHEMA_COLS"
msgstr "Не задано название таблицы для SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:72
#: providers/gda-mysql-server/main-mysql.c:74
msgid "Could not register MySQL provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера MySQL"

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:1
msgid "GDA Datasource Access for MySQL"
msgstr ""

#: providers/gda-mysql-server/GNOME_GDA_Provider_MySQL.oaf.in.h:2
msgid "MySQL provider connection factory"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:175
#, fuzzy
msgid ""
"You cannot provide a UNIX_SOCKET if you also provide either a HOST or a "
"PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT options"
msgstr ""
"%s:%d: Вы не можете указывать UNIX_SOCKET, если вы также указываете Хост или "
"Порт. Пожалуйста удалить параметры Хост и Порт или UNIX_SOCKET"

#: providers/gda-mysql-server/gda-mysql-connection.c:191
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, unix_socket=%"
"s"
msgstr ""
"Открытие соединения с параметрами пользователь=%s, пароль=%s, хост=%s, порт=%"
"s, unix_socket=%s"

#: providers/gda-mysql-server/gda-mysql-connection.c:623
msgid "Unknown"
msgstr ""

#: providers/gda-mysql-server/gda-mysql-connection.c:656
msgid "No MYSQL_Connection for GdaServerConnection"
msgstr ""

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:1
msgid "GDA Datasource Access for ODBC"
msgstr ""

#: providers/gda-odbc-server/GNOME_GDA_Provider_ODBC.oaf.in.h:2
msgid "ODBC provider connection factory"
msgstr ""

#: providers/gda-odbc-server/gda-odbc-connection.c:277
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: необработанный SCHEMA_QTYPE %d\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:641
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-odbc-server/gda-odbc-connection.c:643
#: providers/gda-odbc-server/gda-odbc-connection.c:674
#: providers/gda-odbc-server/gda-odbc-connection.c:675
#: providers/gda-odbc-server/gda-odbc-connection.c:676
msgid "no text"
msgstr "нет текста"

#: providers/gda-odbc-server/main-odbc.c:73
msgid "Could not register ODBC provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера ODBC"

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:1
msgid "GDA Datasource Access for Oracle"
msgstr ""

#: providers/gda-oracle-server/GNOME_GDA_Provider_Oracle.oaf.in.h:2
msgid "Oracle provider connection factory"
msgstr ""

# fe/fe_log.c:92
#: providers/gda-oracle-server/gda-oracle-command.c:191
msgid "Could not allocate statement handle"
msgstr "Не удалось получить обработчик утверждения"

#: providers/gda-oracle-server/gda-oracle-command.c:197
msgid "command object being created without associated connection"
msgstr "командный объект был создан без ассоциированного соединения"

# fe/fe_editor.c:185
#: providers/gda-oracle-server/gda-oracle-command.c:238
#, c-format
msgid "Executing command '%s'"
msgstr "Выполнение команды \"%s\""

#: providers/gda-oracle-server/gda-oracle-connection.c:117
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "ошибка в OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:123
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "ошибка в OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:129
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "ошибка в OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:418
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Неизвестный тип ORACLE %ld"

#: providers/gda-oracle-server/gda-oracle-recordset.c:118
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Неизвестный ORACLE тип данных в fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:156
msgid "recordset object being created without associated connection"
msgstr "объект recordset был создан без ассоциированного соединения"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера ORACLE"

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:1
msgid "GDA Datasource Access for Postgres"
msgstr ""

#: providers/gda-postgres-server/GNOME_GDA_Provider_Postgres.oaf.in.h:2
msgid "Postgres provider connection factory"
msgstr ""

#: providers/gda-postgres-server/gda-postgres-connection.c:888
#: providers/gda-postgres-server/gda-postgres-connection.c:1005
msgid "postgres_recset is NULL"
msgstr "postgres_recset имеет значение NULL"

#: providers/gda-postgres-server/main-postgres.c:71
msgid "Could not register PostgreSQL provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера PostgreSQL"

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:1
msgid "GDA Datasource Access for Sybase"
msgstr ""

#: providers/gda-sybase-server/GNOME_GDA_Provider_Sybase.oaf.in.h:2
msgid "Sybase provider connection factory"
msgstr ""

#.      gda_server_error_make(error, 0, cnc, "connection seems to be dead");
#: providers/gda-sybase-server/gda-sybase-command.c:160
#: providers/gda-tds-server/gda-tds-command.c:121
msgid "Connection seems to be dead"
msgstr "Видимо соединение умерло"

#: providers/gda-sybase-server/gda-sybase-command.c:244
#: providers/gda-tds-server/gda-tds-command.c:149
msgid "Allocating recordset failed"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-command.c:262
#: providers/gda-tds-server/gda-tds-command.c:158
msgid "Getting recordset userdata failed"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-command.c:400
#: providers/gda-tds-server/gda-tds-command.c:242
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() вернуло %x"

#. if (SYB_CHK (NULL,
#. ct_cmd_drop (scmd->cmd),
#. NULL, NULL, cnc, cmd) != CS_SUCCEED) {
#: providers/gda-sybase-server/gda-sybase-command.c:499
#: providers/gda-sybase-server/gda-sybase-command.c:520
#: providers/gda-tds-server/gda-tds-command.c:305
#: providers/gda-tds-server/gda-tds-command.c:315
msgid "Freeing command structure failed"
msgstr "Сбой освобождения структуры команды"

#: providers/gda-sybase-server/gda-sybase-connection.c:182
msgid "Not enough memory for sybase_Connection"
msgstr "Недостаточно памяти для выполнения sybase_Соединения"

#: providers/gda-sybase-server/gda-sybase-connection.c:483
#: providers/gda-tds-server/gda-tds-connection.c:442
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: название таблицы \"%s\""

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:500
#: providers/gda-tds-server/gda-tds-connection.c:459
msgid "Could not allocate command structure"
msgstr "Не удалось получить структуру команды"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:523
#: providers/gda-tds-server/gda-tds-connection.c:482
msgid "Could not allocate enough memory for query"
msgstr "Не удалось получить достаточно памяти для запроса"

#.          gda_server_error_make(error, NULL, cnc, __PRETTY_FUNCTION__);
#: providers/gda-sybase-server/gda-sybase-connection.c:538
#: providers/gda-tds-server/gda-tds-connection.c:497
msgid "Could not get recordset in schema_cols"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-connection.c:587
#: providers/gda-tds-server/gda-tds-connection.c:546
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: недопустимый тип ограничения %d"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:597
#: providers/gda-tds-server/gda-tds-connection.c:556
msgid "Could not allocate cmd structure"
msgstr "Не удалось получить структуру кмд"

#: providers/gda-sybase-server/gda-sybase-connection.c:663
#: providers/gda-tds-server/gda-tds-connection.c:622
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: недопустимый тип ограничения %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:725
#: providers/gda-tds-server/gda-tds-connection.c:684
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Неправильный или неподдерживаемый тип ограничения: %d"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:768
#: providers/gda-tds-server/gda-tds-connection.c:727
msgid "Could not allocate GdaServerCommand"
msgstr "Не удалось получить GdaServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:799
#: providers/gda-sybase-server/gda-sybase-connection.c:804
#: providers/gda-tds-server/gda-tds-connection.c:757
#: providers/gda-tds-server/gda-tds-connection.c:762
#, c-format
msgid "connection status: %x"
msgstr "состояние соединения: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:800
#: providers/gda-tds-server/gda-tds-connection.c:758
#, c-format
msgid "return code: %x"
msgstr "код возврата: %x"

# fe/fe_connection.c:88
#: providers/gda-sybase-server/gda-sybase-connection.c:840
#: providers/gda-tds-server/gda-tds-connection.c:793
msgid "attempting reconnection"
msgstr "попытка пересоединения"

# fe/fe_connection.c:46
#: providers/gda-sybase-server/gda-sybase-connection.c:1030
#, c-format
msgid "Reconnected to '%s'"
msgstr "Пересоединение с \"%s\""

# fe/fe_connection.c:46
#: providers/gda-sybase-server/gda-sybase-connection.c:1032
#: providers/gda-tds-server/gda-tds-connection.c:895
#, c-format
msgid "Connected to %s"
msgstr "Соединен с %s"

# fe/fe_designer.c:40
#: providers/gda-sybase-server/gda-sybase-connection.c:1044
#: providers/gda-tds-server/gda-tds-connection.c:909
#, c-format
msgid "Selected database '%s'"
msgstr "Выбранная база данных \"%s\""

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:1080
#: providers/gda-tds-server/gda-tds-connection.c:935
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Не удалось получить структуру кмд"

#: providers/gda-sybase-server/gda-sybase-connection.c:1090
#: providers/gda-tds-server/gda-tds-connection.c:944
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Не удалось получить память для строки кмд"

# fe/fe_editor.c:138
#: providers/gda-sybase-server/gda-sybase-connection.c:1115
#: providers/gda-tds-server/gda-tds-connection.c:952
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: сбой ct_command"

# fe/fe_editor.c:138
#: providers/gda-sybase-server/gda-sybase-connection.c:1135
#: providers/gda-tds-server/gda-tds-connection.c:962
#, c-format
msgid "%s: sending command failed"
msgstr "%s: отправка команды не удалась"

#: providers/gda-sybase-server/gda-sybase-connection.c:1160
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: сбой выбора базы данных"

# fe/fe_editor.c:138
#: providers/gda-sybase-server/gda-sybase-connection.c:1180
#, c-format
msgid "%s: use command failed"
msgstr "%s: сбой использования команды"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-connection.c:1214
#: providers/gda-tds-server/gda-tds-connection.c:1025
msgid "Could not allocate locale structure"
msgstr "Не удалось получить структуру locale"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-sybase-server/gda-sybase-connection.c:1233
#: providers/gda-tds-server/gda-tds-connection.c:1031
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Не удалось установить локализацию для '%s'"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-sybase-server/gda-sybase-connection.c:1253
#: providers/gda-tds-server/gda-tds-connection.c:1038
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Не удалось настроить локализацию на '%s'"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-sybase-server/gda-sybase-connection.c:1266
#: providers/gda-tds-server/gda-tds-connection.c:1044
msgid "Could not drop locale structure"
msgstr "Не удалось сбросить структуру locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1281
#: providers/gda-tds-server/gda-tds-connection.c:1057
#, c-format
msgid "Fatal error: %s\n"
msgstr "Фатальная ошибка: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:161
msgid "cs_diag() initialized"
msgstr "cs_diag() инициализовано"

#: providers/gda-sybase-server/gda-sybase-error.c:168
msgid "Could not initialize cs_diag()"
msgstr "Не удалось инициализировать cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:185
msgid "ct_diag() initialized"
msgstr "ct_diag() инициализовано"

#: providers/gda-sybase-server/gda-sybase-error.c:193
msgid "Could not initialize ct_diag()"
msgstr "Не удалось инициализировать ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:238
#, c-format
msgid "%s could not find server implementation"
msgstr "%s невозможно найти реализацию сервера"

#: providers/gda-sybase-server/gda-sybase-error.c:246
#, c-format
msgid "%s could not get cnc userdata"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:264
msgid "Could not get # of messages"
msgstr "Не удалось получить число сообщений"

#: providers/gda-sybase-server/gda-sybase-error.c:356
#, fuzzy
msgid "ct_diag failed, Could not get # of client messages(1)"
msgstr "Не удалось получить число клиентских сообщений"

#: providers/gda-sybase-server/gda-sybase-error.c:381
#, fuzzy
msgid "Could not get client msg #"
msgstr "Не удалось получить клиентское сообщение # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:394
#: providers/gda-sybase-server/gda-sybase-error.c:396
#: providers/gda-sybase-server/gda-sybase-error.c:688
#: providers/gda-sybase-server/gda-sybase-error.c:690
#, fuzzy
msgid "Client Message:"
msgstr "Сообщение клиента:"

#: providers/gda-sybase-server/gda-sybase-error.c:424
#, fuzzy
msgid "ct_diag failed, Could not get # of client messages(2)"
msgstr "Не удалось получить число клиентских сообщений"

# fe/fe_editor.c:175 fe/fe_editor.c:212
#: providers/gda-sybase-server/gda-sybase-error.c:442
msgid "Could not clear client messages"
msgstr "Не удалось очистить клиентские сообщения"

#: providers/gda-sybase-server/gda-sybase-error.c:486
#, fuzzy
msgid "Could not get # of server messages(1)"
msgstr "Невозможно получить номер сообщения сервера"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-error.c:510
#, fuzzy
msgid "Could not get server msg #"
msgstr "Не удалось получить серверное сообщение # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:570
msgid "Could not get # of server messages"
msgstr "Невозможно получить номер сообщения сервера"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-error.c:592
msgid "Could not clear server messages"
msgstr "Не удалось очистить серверные сообщения"

#: providers/gda-sybase-server/gda-sybase-error.c:634
msgid "Could not get # of cslib messages"
msgstr "Не удалось получить число сообщений cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:650
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Получение %d сообщений cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:653
#, fuzzy, c-format
msgid "Fetching %d cslib message"
msgstr "Получение %d сообщений cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:672
#, fuzzy
msgid "Could not get cslib msg #"
msgstr "Не удалось получить сообщение cslib # %d"

#: providers/gda-sybase-server/gda-sybase-error.c:711
#, fuzzy
msgid "Could not get # of cslib messages(2)"
msgstr "Не удалось получить число сообщений cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:729
msgid "Could not clear cslib messages"
msgstr "Не удалось очистить сообщения cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:761
msgid "SYB"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:763
msgid "severity"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:765
msgid "number"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:767
msgid "origin"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:769
msgid "layer"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-error.c:776
#, c-format
msgid ""
"\n"
"\tOperating system error number(%ld):\n"
"\t%s"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:219
#: providers/gda-tds-server/gda-tds-recordset.c:199
#, c-format
msgid "fetch error at row %d"
msgstr "получена ошибка в строке %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:226
#: providers/gda-tds-server/gda-tds-recordset.c:206
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA в строке %d; отмена"

#: providers/gda-sybase-server/gda-sybase-recordset.c:251
#: providers/gda-tds-server/gda-tds-recordset.c:213
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Ошибка (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:473
#: providers/gda-tds-server/gda-tds-recordset.c:387
msgid "Failed fetching # of columns"
msgstr "Сбой получения числа столбцов"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-recordset.c:494
#: providers/gda-tds-server/gda-tds-recordset.c:402
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s Не удалось получить информацию о формате данных"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-recordset.c:507
#: providers/gda-tds-server/gda-tds-recordset.c:412
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s Не удалось получить поля данных"

#: providers/gda-sybase-server/gda-sybase-recordset.c:525
#: providers/gda-tds-server/gda-tds-recordset.c:430
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe на столбце # %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:532
#: providers/gda-tds-server/gda-tds-recordset.c:437
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "сбой ct_describe на столбце # %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:539
#: providers/gda-tds-server/gda-tds-recordset.c:444
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): \"%s\", %d (%s)"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-recordset.c:551
#: providers/gda-tds-server/gda-tds-recordset.c:455
msgid "could not allocate data holder"
msgstr "не удалось получить место для хранения данных"

#: providers/gda-sybase-server/gda-sybase-recordset.c:564
#: providers/gda-tds-server/gda-tds-recordset.c:468
msgid "could not ct_bind data holder to column"
msgstr ""

#: providers/gda-sybase-server/gda-sybase-recordset.c:571
#: providers/gda-tds-server/gda-tds-recordset.c:475
msgid "illegal type detected, aborting"
msgstr "обнаружен недопустимый тип, прерывание"

# fe/fe_log.c:92
#: providers/gda-sybase-server/gda-sybase-recordset.c:578
#: providers/gda-tds-server/gda-tds-recordset.c:482
msgid "could not allocate field"
msgstr "не удалось получить поле"

#: providers/gda-sybase-server/main-sybase.c:78
msgid "Could not register gda-sybase provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера gda-sybase"

#: providers/gda-primebase-server/GNOME_GDA_Provider_Primebase.oaf.in.h:1
msgid "GDA Datasource Access for primebase"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:157
#, c-format
msgid "Could not connect to '%s@%s:%s' with connparm '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:163
#, c-format
msgid "%2ld: Connected to '%s@%s:%s' with connparm: '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:171
msgid "Could not get PBIConnectionInfo(), disconnecting"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:176
#, c-format
msgid "%2ld: ConInfo: '%s@%s', Cl v%x, Srv '%s' v%x, Brand '%s'"
msgstr ""

#: providers/gda-primebase-server/gda-primebase-connection.c:390
#, fuzzy
msgid "Not enough memory for primebase_Error structure."
msgstr "Недостаточно памяти для выполнения sybase_Соединения"

#: providers/gda-primebase-server/gda-primebase-connection.c:397
#, c-format
msgid "%2ld: ERROR: %ld (%ld) : %s.\n"
msgstr ""

#: providers/gda-primebase-server/main-primebase.c:73
#, fuzzy
msgid "Could not register PRIMEBASE provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера INTERBASE"

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:1
msgid "GDA Datasource Access for Sybase/MS SQL"
msgstr ""

#: providers/gda-tds-server/GNOME_GDA_Provider_TDS.oaf.in.h:2
msgid "TDS provider connection factory"
msgstr ""

#: providers/gda-tds-server/gda-tds-command.c:135
msgid "Filling command structure failed\n"
msgstr "Сбой заполнения управляющей структуры\n"

#: providers/gda-tds-server/gda-tds-command.c:142
msgid "sending cmd failed"
msgstr "сбой отправки кмд"

#: providers/gda-tds-server/gda-tds-command.c:215
msgid "Command execution finished, all results processed"
msgstr "Выполнение команды завершено, все результаты обрабатываются"

# fe/fe_editor.c:138
#: providers/gda-tds-server/gda-tds-command.c:219
msgid "Command failed"
msgstr "Выполнение команды не удалось"

#: providers/gda-tds-server/gda-tds-command.c:224
msgid "Execution of resultless command succeeded"
msgstr "Выполнение \"безрезультатной\" команды прошло успешно"

#: providers/gda-tds-server/gda-tds-command.c:254
msgid "No further results, command completly succeeded."
msgstr "Нет последующих результатов, команда выполнена успешно."

#: providers/gda-tds-server/gda-tds-command.c:259
msgid "An error occured during command execution."
msgstr "При выполнении команды возникла ошибка."

#: providers/gda-tds-server/gda-tds-command.c:263
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"Выполнение команды привело к неизвестному результату, возможно к ошибке"

#: providers/gda-tds-server/gda-tds-command.c:274
msgid "No data returned, freeing recordset structure"
msgstr "Данные не были возвращены, освобаждается структура записи"

#: providers/gda-tds-server/gda-tds-connection.c:165
#, fuzzy
msgid "Not enough memory for tds_Connection"
msgstr "Недостаточно памяти для выполнения sybase_Соединения"

# fe/fe_connection.c:46
#. Fixme: replace "dummy" with buf when CS_GET CS_SERVERNAME works
#: providers/gda-tds-server/gda-tds-connection.c:893
#, fuzzy, c-format
msgid "Reconnected to %s"
msgstr "Пересоединение с \"%s\""

#: providers/gda-tds-server/gda-tds-error.c:75
#, fuzzy, c-format
msgid "gda_tds_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-tds-server/gda-tds-error.c:86
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""

#: providers/gda-tds-server/gda-tds-error.c:274
msgid "Server message:"
msgstr "Сообщение сервера:"

#: providers/gda-tds-server/gda-tds-error.c:282
msgid "Client message:"
msgstr "Сообщение клиента:"

#: providers/gda-tds-server/gda-tds-error.c:289
msgid "CS-Library error:"
msgstr "Ошибка CS-библиотеки:"

#: providers/gda-tds-server/gda-tds-recordset.c:395
#, c-format
msgid "Counted %d columns"
msgstr "Подсчитано %d столбцов"

#: providers/gda-tds-server/main-tds.c:78
#, fuzzy
msgid "Could not register gda-tds provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера gda-sybase"

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: неверная запись DSN"

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: источник данных \"%s\" не найден\n"

#: tools/templates/main-srv.c.tmpl:60
#, fuzzy
msgid "Could not register <PROVIDER-NAME> provider implementation"
msgstr "Не удалось зарегистрировать реализацию провайдера INTERBASE"

#~ msgid "gda_sybase_error_make(%p,%p,%p,%p)"
#~ msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#~ msgid "Fetching %d client messages"
#~ msgstr "Получение %d клиентских сообщений"

#~ msgid "Fetching %d client message"
#~ msgstr "Получение %d клиентского сообщения"

#~ msgid "Fetching %d server messages"
#~ msgstr "Получение %d серверных сообщений"

#~ msgid "Fetching %d server message"
#~ msgstr "Получение %d серверного сообщения"

# gda-clnt/gda-field.c:117
#~ msgid "TRUE"
#~ msgstr "ИСТИНА"

# gda-clnt/gda-field.c:119
#~ msgid "FALSE"
#~ msgstr "ЛОЖЬ"

#~ msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
#~ msgstr "impl_GDA_Connection_open: открытие соединения с DSN: %s"

#~ msgid "%s: an error ocurred while fetching data"
#~ msgstr "%s: обнаружена ошибка при получении данных"

#~ msgid ""
#~ "OAF doesn't know about our IID; indicates broken installation. Exiting..."
#~ msgstr ""
#~ "OAF не знает о нашем IID; указывает на неправильную установку. Выход..."

#~ msgid ""
#~ "Another instance of this provider is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Другой экземпляр этого провайдера уже зарегистрирован в OAF. Выход..."

#~ msgid "Unknown error registering this provider with OAF. Exiting"
#~ msgstr "Неизвестная ошибка регистрации этого провайдера в OAF. Выход"

#~ msgid "Registered with ID = %s"
#~ msgstr "Зарегистрировано с идентификатором = %s"

#~ msgid "CORBA exception unloading report engine: %s"
#~ msgstr "Исключение CORBA при выгрузке процессора отчетов: %s"

#~ msgid "Received signal %d, shutting down."
#~ msgstr "Получен сигнал %d, выключение."

#~ msgid "Received signal %d, shutting down cleanly"
#~ msgstr "Получен сигнал %d, чистое выключение"

#~ msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
#~ msgstr ""
#~ "Сбой инициализации OAF: пожалуйста, отправьте сообщение об ошибке "
#~ "разработчикам OAF"

# fe/fe_log.c:92
#~ msgid "Could not activate report engine"
#~ msgstr "Не удалось активировать процессор отчетов"

#~ msgid ""
#~ "Another instance of the report engine is already registered with OAF. "
#~ "Exiting..."
#~ msgstr ""
#~ "Другой экземпляр процессора отчетов уже зарегистрирован в OAF. \n"
#~ "Выход..."

#~ msgid "Unknown error registering report engine with OAF. Exiting"
#~ msgstr ""
#~ "Неизвестная ошибка регистрации процессора отчетов в OAF. \n"
#~ "Выход"

#~ msgid "Registered report engine with ID = %s"
#~ msgstr "Зарегистрирован процессор отчетов с идентификатором = %s"

#~ msgid "%s: length = %d"
#~ msgstr "%s: длина = %d"

#~ msgid "%s: sending field description for '%s'"
#~ msgstr "%s: послано описание поля для \"%s\""

#~ msgid "Error allocating error structure"
#~ msgstr "Ошибка получения структуры ошибки"

#~ msgid "schema_tables: table name = '%s'"
#~ msgstr "schema_tables: название таблицы = \"%s\""

#~ msgid "schema_tables: table_schema = '%s'"
#~ msgstr "schema_tables: table_schema = \"%s\""

#~ msgid "Fetchong %d cslib message"
#~ msgstr "Получение %d сообщения cslib"

# fe/fe_main.c:186
#~ msgid "database name"
#~ msgstr "название базы данных"

#~ msgid "DATABASE"
#~ msgstr "БАЗА ДАННЫХ"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#~ msgid "user name"
#~ msgstr "имя пользователя"

#~ msgid "USER-NAME"
#~ msgstr "ИМЯ ПОЛЬЗОВАТЕЛЯ"

#~ msgid "password"
#~ msgstr "пароль"

#~ msgid "PASSWORD"
#~ msgstr "ПАРОЛЬ"

# fe/fe_impexp.c:164 fe/fe_impexp.c:274 fe/fe_impexp.c:391
# gda-clnt-ui/gnome-db-browser.c:217
#~ msgid "file name"
#~ msgstr "название файла"

#~ msgid "FILE-NAME"
#~ msgstr "НАЗВАНИЕ ФАЙЛА"
